src/app/lib/utilities/getPassportHome.js

```diff
@@ -0,0 +1,14 @@
+import deepGet from './deepGet';
```

Comment:
Can you organise this file + its respective test file as follows, to keep it consistent with our folder structure:

```
src/app/lib/utilities/getPassportHome/index.js
src/app/lib/utilities/getPassportHome/index.test.js
```